A100-client-side-weighted-round-robin-slow-start.md

```diff
@@ -194,55 +240,73 @@ message SlowStartConfig {
 }
 ```
 
+xDS PR: https://github.com/envoyproxy/envoy/pull/40090
+
+### Metrics
+
+The following metric will be exposed to help monitor the slow start behavior:
+
+`grpc.lb.wrr.endpoints_in_slow_start`
+
+- Type: Counter
+- Description: Number of endpoints currently in slow start period
```

Comment:
Please define exactly when the counter will be incremented.  I assume this will happen every time we construct a new scheduler?